static/js/basic_table_filter.js

```diff
@@ -0,0 +1,31 @@
+document.addEventListener("DOMContentLoaded", function () {
+  const propertyField = document.getElementById("id_property");
+  const unitField = document.getElementById("id_unit");
+
+  function updateDropdown(url, field, placeholder) {
+    fetch(url)
+      .then((response) => response.json())
+      .then((data) => {
+        console.log("Received data:", data);
```

Comment:
console.log are for debugging, never put them in final code until they serve some feature purpose.